src/mjolnir/pbfgraphparser.cc

```diff
@@ -1995,13 +1995,15 @@ OSMData PBFGraphParser::ParseWays(const boost::property_tree::ptree& pt,
   LOG_INFO("Finished");
 
   // Return OSM data
+  osmdata.initialized = true;
   return osmdata;
 }
 
 void PBFGraphParser::ParseRelations(const boost::property_tree::ptree& pt,
                                     const std::vector<std::string>& input_files,
                                     const std::string& complex_restriction_from_file,
                                     const std::string& complex_restriction_to_file,
+                                    const std::string& tile_dir,
```

Comment:
we already have this in the `pt` above can we just use it from there instead

Reply:
@kevinkreiser updated.  thanks for the suggestions.